liblcdf/filename.cc

```diff
@@ -126,5 +126,5 @@ Filename::open_write(bool binary) const
     if (_actual || !_path)
 	return _actual;
     else
-	return fopen(_path.c_str(), binary ? "wb" : "w");
+	return fopen(_path.c_str(), "wb");
```

Comment:
Is this really necessary?

Reply:
Seems like you're misreading the patch. In its current state the function is **not** ignoring the `binary` argument. The patch would make it ignore the argument.